python/kserve/kserve/protocol/rest/v2_datamodels.py

```diff
@@ -229,10 +229,11 @@ class InferenceResponse(BaseModel):
         }
     """
     model_name: str
-    model_version: Optional[str]
+    model_version: Optional[str] = None
     id: str
-    parameters: Optional[Parameters]
+    parameters: Optional[Parameters] = None
     outputs: List[ResponseOutput]
+    model_config = ConfigDict(protected_namespaces='')
 
     class Config:
```

Comment:
This is pydantic 1 syntax.

We should move the `schema_extra` into the `json_schema_extra` field in the `model_config` above.

https://docs.pydantic.dev/latest/migration/#changes-to-config